apisix/admin/stream_routes.lua

```diff
@@ -25,23 +27,69 @@ local _M = {
     need_v3_filter = true,
 }
 
+local function check_router_refer(items, id)
+    local refer_list = {}
+    local referkey = "/stream_routes_refer/" .. id
+    local _, err = core.etcd.delete(referkey)
+    core.log.warn(err)
```

Comment:
Is the `err` here only printed and not processed?

Reply:
deleting a stream route, check if it is referenced by another stream route ; before query ，delete previous value and then generate new。 Just delete previous ,  I choose to ignore exceptions handle  because  I can't think of errors happening